CHANGELOG.md

```diff
@@ -5,6 +5,30 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Added `sensor` field to `OpLog`.
+  - Added the `load_connection_by_prefix_timestamp_key` function and
+    `TimestampKeyExtractor` trait to enable querying of keys prefixed with
+    `timestamp`.
+  - The `opLogRawEvents` GraphQL API no longer requires `agentId` and now
+    accepts it as an optional parameter. Additionally, the API response now
+    includes logs from all agents displayed in chronological order, rather than
+    being limited to the logs of a single agent.
+
+### Changed
+
+- Update the compatibility version of the quic communication modules.
+  - Changed `PEER_VERSION_REQ` to ">=0.24.0-alpha.1,<0.25.0".
+  - Changed `INGEST_VERSION_REQ` to ">=0.24.0-alpha.1,<0.25.0".
+  - Changed `PUBLISH_VERSION_REQ` to ">=0.24.0-alpha.1,<0.25.0".
+- Fixed code related to migration.
```

Comment:
If this was a fix, I would like to suggest locating this entry in the Fixed section of Changelog. But I think this is not a fix but just a change, I would like to suggest that we use words such as change or modify.